core/src/components/input/test/slot/index.html

```diff
@@ -91,8 +91,49 @@ <h2>Outline / Floating</h2>
               <div slot="label">Email <span class="required">*</span></div>
             </ion-input>
           </div>
+
+          <div class="grid-item">
+            <h2>Outline / Floating / Async</h2>
+            <ion-input id="solid-async" label-placement="floating" fill="outline" value="hi@ionic.io"></ion-input>
```

Comment:
If I update the slotted content to the really long label, then remove it, the notch does not re-calculate, should it?
![Screenshot 2023-06-14 at 1 14 19 PM](https://github.com/ionic-team/ionic-framework/assets/6577830/19f81963-d64a-443e-94b7-5f60d1f868a5)

When it starts out with no label the notch is much smaller:
![Screenshot 2023-06-14 at 1 12 48 PM](https://github.com/ionic-team/ionic-framework/assets/6577830/38c2f5d8-234c-42bf-a331-5b7494be724b)

Reply:
Removing the slotted content was fixed in https://github.com/ionic-team/ionic-framework/pull/27636/commits/db1d32e33e74175447814bc8a242f4fa0ee71fa0

The small notch with no label is a bug in `main` too, but I fixed it in https://github.com/ionic-team/ionic-framework/pull/27636/commits/7d2ea9dca4985efa0c343cefbdf4fa2b6a988b75

Reply:
I was actually going to comment that the small notch showing without a label was a bug too but I looked and Material Design has it too. 😂 I do think it's better to just hide it though so this is an improvement.

Reply:
Actually the small notch thing also impacts textarea and select, so I'm going to fix that in `main` instead.

Reply:
Makes sense to me. I'm fine with leaving it separate.

Reply:
Textarea fix: https://github.com/ionic-team/ionic-framework/pull/27647/commits/178cdebb8fa0525554ddbc193219cb93434b2e81i

Select fix: https://github.com/ionic-team/ionic-framework/pull/27649